pkg/console/operator/operator.go

```diff
@@ -200,21 +208,33 @@ func (c *consoleOperator) deleteAllResources(cr *consolev1.Console) error {
 	return utilerrors.FilterOut(utilerrors.NewAggregate(errs), errors.IsNotFound)
 }
 
-// this may need to eventually live under each sync version, depending on if there is
-// custom sync logic
-func (c *consoleOperator) defaultConsole() *consolev1.Console {
-	logrus.Info("creating console CR with default values")
-	return &consolev1.Console{
+func (c *consoleOperator) defaultConsoleConfig() *configv1.Console {
+	return &configv1.Console{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: api.ResourceName,
+		},
+		Spec: configv1.ConsoleSpec{
+			//Authentication: consoleconfigv1.ConsoleAuthentication{
```

Comment:
Default should not populate this value, but its the one thing we need to honor if set.

Reply:
Let's remove the commented out code if it's not needed